src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -616,7 +616,7 @@ private List<FeedbackResponseAttributes> buildMissingResponses(
         return missingResponses;
     }
 
-    private boolean isResponseVisibleForUser(
+    boolean isResponseVisibleForUser(
```

Comment:
@wkurniawan07 suggested to change this to `protected` but when I change to `protected`, IntelliJ says it's `"Class member declared 'protected' in 'final' class"`. If protected is indeed better, feel free to let me know again!

Reply:
I am very certain my suggestion is package level, which is exactly what has been done here

Reply:
Ahh thanks. It seems like I misread in that case. Thanks!